single_kernel_mongo/events/lifecycle.py

```diff
@@ -164,6 +164,8 @@ def on_relation_changed(self, event: RelationChangedEvent):
             event.defer()
             return
 
+        self.charm.status_manager.process_and_share_statuses()
```

Comment:
you sure you want this here?

Reply:
I guess we discussed about it in another PR, we'll see how it goes but for now it's quite helpful and reports statuses faster (for example, no need to wait until the first `update-status` to know we're missing an integration if we're a config-server or a shard.
If something goes south, it'll be removed.

Reply:
Great! thats fine by me! s